pkgs/development/python-modules/importlib-metadata/default.nix

```diff
@@ -27,6 +27,11 @@ buildPythonPackage rec {
     ++ lib.optionals (!isPy3k) [ pathlib2 contextlib2 configparser ];
 
   checkInputs = [ importlib-resources packaging ];
+  preCheck = ''
+    # removing test_main.py - it requires 'pyflakefs'
+    # and adding `pyflakefs` to `checkInputs` causes infinite recursion.
```

Comment:
I would move the comments outside of the string, if we want to edit the comments later, the string comments will cause a rebuild, while nix comments will not

Reply:
makes sense, thx. addressed and pushed already.